Amazon.QLDB.Driver/client/Session.cs

```diff
@@ -270,5 +314,10 @@ private SendCommandResponse SendCommand(SendCommandRequest request)
             this.logger.LogDebug("Sending request: {}", request);
             return this.SessionClient.SendCommandAsync(request).GetAwaiter().GetResult();
```

Comment:
1) Remove GetAwaiter().GetResult() from here. Make this function async and return `Task<SendCommandResponse>`

2) Rename current sync methods to async, mark them as async, and await/return the tasks

3) Create new sync functions, that just call the async functions, and call .GetAWaiter().GetResult() on them.